scout/server/blueprints/cases/controllers.py

```diff
@@ -388,6 +390,48 @@ def coverage_report_contents(store, institute_obj, case_obj, base_url):
     return coverage_data
 
 
+def mt_coverage_stats(individuals, ref_chrom="21"):
+    """Send a request to chanjo report endpoint to retrieve MT vs autosome coverage stats
+
+    Args:
+        individuals(dict): case_obj["individuals"] object
+        ref_chrom(str): reference chromosome (1-22)
+
+    Returns:
+        coverage_stats(dict): a dictionary with mean MT and autosome transcript coverage stats
+    """
+    coverage_stats = {}
+    ind_ids = []
+    for ind in individuals:
+        ind_ids.append(ind["individual_id"])
+
+    # Prepare complete url to Chanjo report chromosome mean coverage calculation endpoint
+    cov_calc_url = url_for("report.json_chrom_coverage", _external=True)
```

Comment:
Oh, is external needed? Nothing wrong, just a little surprised. Well found then!

Reply:
Yes, otherwise you get an error when you create the url (it tells you that the url can't be created, did you mean blabla ? *blabla = some other random view)